docs/src/userguide/language_basics.rst

```diff
@@ -568,6 +570,9 @@ possibilities.
    combinations.  An example is a union of an ``int`` and a ``char*``,
    in which case the pointer value may or may not be a valid pointer.
 
+.. [#2] The conversion will fail if the C array is not stack allocated,
+   or when using a slice of a C array.
```

Comment:
That sounds a lot like a missing feature.

Reply:
This makes me think that I should link missing features (in Github issues) to the part of the docs where it's mentioned. So that we don't forget (and that it's easier) to update the docs once the feature is implemented.